extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMusicArtistInfoItemExtractor.java

```diff
@@ -61,10 +62,14 @@ public String getUrl() throws ParsingException {
 
     @Override
     public long getSubscriberCount() throws ParsingException {
-        final String subscriberCount = getTextFromObject(artistInfoItem.getArray("flexColumns")
-                .getObject(2)
+        final JsonArray flexColumns = artistInfoItem.getArray("flexColumns");
+        final JsonArray runs = flexColumns
+                .getObject(flexColumns.size() - 1)
                 .getObject("musicResponsiveListItemFlexColumnRenderer")
-                .getObject("text"));
+                .getObject("text")
+                .getArray("runs");
+        final String subscriberCount = runs.getObject(runs.size() - 1)
+                .getString("text");
```

Comment:
I think you can use YoutubeParsingHelper.getTextFromObject()

Reply:
That was used there before however I did not choose to use it here.

runs returns an array with 3 items containing:
0: <artistname>
1: •
2: <subscriperCount>

There is 2 problems with that:
1. It's inefficient
2. If for some reason the artist now has a name like "2k" the extraction malfunctions
 
I can add this as a comment to the code if you wish ;)

Reply:
Added the comment 👍